Makefile.config.example

```diff
@@ -94,6 +94,16 @@ LIBRARY_DIRS := $(PYTHON_LIB) /usr/local/lib /usr/lib
 # INCLUDE_DIRS += $(shell brew --prefix)/include
 # LIBRARY_DIRS += $(shell brew --prefix)/lib
 
+# NCCL acceleration switch (uncomment to build with NCCL)
+# E.g. setup:
+#   cd
+#   git clone https://github.com/NVIDIA/nccl
+#   cd nccl
+#   make -j
+# USE_NCCL := 1
+# INCLUDE_DIRS += $(HOME)/nccl/src
```

Comment:
If they install it its the default /usr/local path, so they don't need that. I could remove this, not sure if installing would be the typical setup for nccl.